### DISCUSSION TITLE
NexcloudPi update

### DISCUSSION THREAD

**Discussion by eteixido1** [tteck/Proxmox#3158] [Help Needed]
Hello @tteck and team,
I don't know if it's my fault or not but when I try to update my nextcloudpi with the update scripts the whole "recognize" tables are wiped. I tryed some times and it happens each time I tryed it. Maybe it isn't the update script but its really annoying because I have a lot of people classyfied and maybe this could happen to other people.

Kind regards,

**Comment by Mellowlynx** [comment]
What “update scripts” are you revering too?
For NexcloudPi there is no update script made by ttack, it should be updated through the NexcloudPi WebUI on port 4443 (so not the Nextcloud UI), or trough ssh `sudo ncp-config`.

I do not have any issues with wiped tables, running updates this way.

**Reply by tteck** [reply]
@Mellowlynx thank you for all your recent assistance in answering questions. 🙏🏻

**Reply by Mellowlynx** [reply]
@tteck You are very welcome.

**Reply by eteixido1** [reply]
Hello @Mellowlynx ,
I used standard lxc update script used to install and update as other helper scripts.
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/ct/nextcloudpi.sh)"

I tryed too the Proxmox VE LXC updater:
bash -c "$(wget -qLO - https://github.com/tteck/Proxmox/raw/main/misc/update-lxcs.sh)"

And the problem is the same. Maybe I have a wrong configuration but using wichever of them I lose all "recognize" faces inside nextcloud.

Kind regards,

**Reply by tteck** [reply]
Nowhere in the instructions does it mention to update in this manner.

![Screenshot 2024-06-10 7 24 53 AM](https://github.com/tteck/Proxmox/assets/21342670/a9b903bc-6d8f-4a3d-90f7-0f7a19ffc9f6)

Both of the scripts that you are executing, are doing a `apt-get update && apt-get upgrade` which shouldn't affect Nextcloud.
